projects/azure/README.md

```diff
@@ -0,0 +1,25 @@
+# Azure project
+
+Azure project is flavor of OpenShift dedicated hosted on Microsoft Azure. This repository contains code for building release artefacts, testing, and life-cycle.
+Main code repository is located in [Openshift Azure ](https://github.com/openshift/openshift-azure/) project
+
+# Test CI-operator jobs 
+
+CI-Operator jobs are being triggered using [prow](https://github.com/kubernetes/test-infra/tree/master/prow)
+Prow configuration is located in this repository `cluster/ci/config/prow/config.yaml`
+
+To run CI-Operator job manually you need to have to have (CI-Operator)[https://github.com/openshift/ci-operator] installed in your path.
+Modify secret location in file `cluster-launch-e2e-azure.yaml` as beflow. This is because `ci-operator` set secret based on path where files are located and they are different in local development and CI server.
+``` - name: cluster-secrets-azure
+      secret:
+    #    secretName: e2e-azure-secret
+         secretName: azure
+```
+
+Populate secret file in `cluster/test-deploy/azure/secret` using `cluster/test-deploy/azure/secret_example` as template.
+If you running this job not in CI cluster development namespace, but on other Openshift cluster you will need to pre-populate base images in `azure` namespaces. 
```

Comment:
If you are running

Reply:
I think it's fine to omit this sentence altogether. The azure namespace is public and is meant to be accessible as long as the CI cluster is up and running.